www/kanban/inner.js

```diff
@@ -10,6 +10,7 @@ define([
     '/common/modes.js',
     '/customize/messages.js',
     '/kanban/jkanban.js',
+    '/kanban/jscolor.js',
```

Comment:
I'm curious, what does jscolor do that the native colorpickers can't? I hadn't realized that this PR pulled in a library until now, and we'll have to do a bit of a review of its code before pulling it in.

Reply:
The most important thing it can do instead of native color pickers is that (atleast on windows) the native color picker is a popup window. Which in my opinion does a better workflow in an app like this. (A popup feels more disctracting than this "overlay popup")

Their page is here: http://jscolor.com/

Reply:
Thanks! I agree about the overlay being less distracting. I'll put some time towards reading through their source, but as long as there's nothing that poses a security risk, I'm in favour of pulling it in.